service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -66,7 +67,7 @@ public static function fromArray(mixed $data): self
         $instance->personType = $data['personType'];
         $instance->firstName = $data['firstName'];
         $instance->lastName = $data['lastName'];
-        $instance->dob = $data['dob'];
+        $instance->dob = $data['dob'] ?? 'NA';
```

Comment:
If the field is nullable, this should fallback to null rather than a string
```suggestion
        $instance->dob = $data['dob'] ?? null;
```

Reply:
My initial attempt was to have this as null as above, however it seems while this satisfies validator, dynanomdb does not like a null value, but can't find a reason why it wouldn't. Will dig abit more into this